src/commands/download_file.ts

```diff
@@ -0,0 +1,74 @@
+import { EID } from 'ardrive-core-js';
+import { Stats, statSync } from 'fs';
+import { join as joinPath, parse as parsePath, resolve as resolvePath } from 'path';
+import { cliArDriveAnonymousFactory, cliArDriveFactory } from '..';
+import { CLICommand, ParametersHelper } from '../CLICommand';
+import { CLIAction } from '../CLICommand/action';
+import {
+	DriveIdParameter,
+	DrivePrivacyParameters,
+	DryRunParameter,
+	FileIdParameter,
+	LocalFilePathParameter
+} from '../parameter_declarations';
+
+/**
+ * Gets the full file name
```

Comment:
Needs better documentation. What's considered a `full file name`? What's the purpose of all the other parameters to fulfilling the goal of the function?

Reply:
Since there's no tests showing how this function works, can you provide some examples, given a fictional folder/file tree, of how this function would perform with different inputs?